src/components/crown/crownButtons/crownColorDropdown.vue

```diff
@@ -82,18 +125,44 @@ export default {
     setColorFromPicker({ hex8 }) {
       this.selectedColor = hex8;
     },
-    unsetNodeColor() {
-      store.commit('updateNodeProp', { node: this.node, key: 'color', value: undefined });
-      this.$emit('save-state');
+    resetNodeStyling() {
+      this.resetNodeColor();
+      this.resetCustomIcon();
     },
     setNodeColor(color) {
-      Vue.set(this.node.definition, 'color', color);
       store.commit('updateNodeProp', { node: this.node, key: 'color', value: color });
+      Vue.set(this.node.definition, 'color', color);
```

Comment:
Setting the property so it's reactive to Vue may already happen in the `updateNodeProp` store mutation and might not be required here.

Reply:
`Vue.set` cannot be safely removed until the `pm:customIcon` property has been added to the BPMN extensions.